itwin-platform-access/imodels-access-frontend/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Change Log - @itwin/imodels-access-frontend
+
+## 2.0.0
+
+This release contains no changes compared to the previous package version.
```

Comment:
Yes, since usually whoever references `@itwin/imodels-acess-*` also references `@itwin/imodels-client-*` I think it would be nice to keep the major versions consistent.

Reply:
ok this makes sense, i guess it'll be easier for consumers to update pkgs with `@itwin/imodels-acess-*` in lockstep.